File: 1013/1013 - O Maior.cpp
#include <iostream>
#include <cstdlib>
int main() {
    int a, b, c;
    std::cin >> a; std::cin >> b; std::cin >> c;
    int maior = (a + b + abs(a-b))/2;
    int maior2 = (maior + c + abs(maior-c))/2;
    std::cout << maior2 << " eh o maior" << std::endl;
    return 0;
}

File: 1013/1013 - O Maior.ex
[a, b, c] = IO.read(:stdio,:line) |> String.trim |> String.split(" ") |> Enum.map(fn str ->
  case Integer.parse(str) do
    {number, _} -> number
  end
end)

maior = div (a + b + abs(a - b)), 2
maior2 = div (maior + c + abs(maior - c)), 2
IO.puts "#{maior2} eh o maior"
